assessment/asr_validation/wald2017B/doc/wald2017B.md

```diff
@@ -25,16 +25,17 @@ Two of these experiments are included here as assessment cases: one unreinforced
 
 !listing test/tests/concrete_ASR_validation/tests
 
-!listing wald2017B/analysis/assessment
+!listing wald2017B/analysis/calibration/assessment
+!listing wald2017B/analysis/rebar/assessment
 
 
 ## Input Files
 
 !listing test/tests/concrete_ASR_validation/asr_validation.i
 
-!listing wald2017B/analysis/asr_concrete_block_calibration.i
+!listing wald2017B/analysis/calibration/asr_concrete_block_calibration.i
 
-!listing wald2017B/analysis/asr_concrete_block_validation_rebar.i
+!listing wald2017B/analysis/rebar/asr_concrete_block_validation_rebar.i
```

Comment:
Can we rename this directory something else other than `rebar`? How about `validation` instead? We can keep all the related future validation cases here.

Reply:
Well, we're going to need separate directories for each one of the rebar sub-cases. I think you're going to be adding a bunch of other rebar cases, and I figured that when you do that, you would probably want to re-name this one to be more specific to uniaxial rebar. We could either do that then, or re-name it now. What name would you give it given that this is the uniaxial rebar case?

Reply:
I was thinking 'validation/A1-001', based on the corresponding experimental case. But we can leave it for now, I can reorganize the directories in my PR.